content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -0,0 +1,228 @@
+      dataType: "histogram"
+      aggregations:
+        - label: "path"
+          displayName: "Path"
+        - label: "method"
+          displayName: "Method"
+    - name: "Server active connections"
+      unit: ""
+      spans: 6
+      metricName: "vertx_http_server_connections"
+      dataType: "raw"
+```
+
+The *name* field (from metadata) corresponds to what you will have to set in pods annotation `kiali.io/runtimes`.
+
+Spec fields definitions are:
+
+* *runtime*: name of the related runtime. It will be displayed on the corresponding Workload Details page. Can be omitted to not display anything.
+* *title*: dashboard title, displayed as a tab in App or Workloads Details
+* *charts*: list of charts in the dashboard
+** *name*: name of the chart
+** *unit*: unit for Y-axis. This unit may be converted using metric SI system, for instance "s" can be shown as "ms" or "µs" as well.
+** *spans*: number of "spans" taken by the chart, from 1 to 12, using link:https://www.w3schools.com/bootstrap4/bootstrap_grid_system.asp[bootstrap convention]
+** *metricName*: the metric name in Prometheus
+** *dataType*: type of data to be displayed in the chart. Can be one of _raw_, _rate_ or _histogram_. Raw data will be queried without transformation. Rate data will be queried using link:https://prometheus.io/docs/prometheus/latest/querying/functions/#rate[_promQL rate() function_]. And histogram with link:https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile[_histogram_quantile() function_].
+** *aggregator*: defines how the time-series are aggregated when several of them are returned for a given metric and labels set. For example, if a Deployment creates a ReplicaSet of several Pods, you will have at least one time-series per Pod. Since Kiali shows the dashboards at the Workload (ReplicaSet) level or at the App level, they will have to be aggregated. This field can be used to fix the aggregator, with values such as _sum_ or _avg_ (full list available link:https://prometheus.io/docs/prometheus/latest/querying/operators/#aggregation-operators[in Prometheus documentation]). However, you can omit it: in that case, the aggregator will default to _sum_ and can be changed from the dashboard UI.
+** *aggregations*: list of labels eligible for aggregations / groupings (they will be displayed in Kiali through a dropdown list)
+*** *label*: Prometheus label name
+*** *displayName*: Name to display in Kiali
+
+Dashboard resources are added in Kubernetes just like any other resource:
+
+```bash
+kubectl apply -f mydashboard.yml
+```
+
+Or for OpenShift:
+
+```bash
+oc apply -f mydashboard.yml
+```
+
+To make them available system-wide, just create them in Kiali namespace (usually _istio-system_). Else, they will be available only for Apps or Workloads of the same namespace. In the case where the same dashboard name exists in a specific namespace and in Kiali namespace, the former takes precedence.
```

Comment:
Shouldn't at the end "Kiali ns" be "istio-system" ? Or if you make this 'location independent', then change istio-system above to Kial-ns

Reply:
I don't understand your remark. Technically speaking the namespace to install cluster-wide dashboard is Kiali/Istio's own namespace, but I just specified that this namespace is usually "istio-system", unless configured differently. So it seems correct to me to refer to Kiali namespace.